src/FFI.php

```diff
@@ -163,9 +163,9 @@ public static function version(): string
      */
     public static function atLeast(int $x, int $y, int $z = 0): bool
     {
-        return self::$library_major > $x ||
-            self::$library_major == $x && self::$library_minor > $y ||
-            self::$library_major == $x && self::$library_minor == $y && self::$library_micro >= $z;
+        return self::$library_major > $x
+            || (self::$library_major === $x && self::$library_minor > $y)
+            || (self::$library_major === $x && self::$library_minor === $y && self::$library_micro >= $z);
```

Comment:
I don't think this fixes (or breaks) anything, but at least it should be clearer now.